src/combat.ts

```diff
@@ -11,17 +12,30 @@ export default class Macro extends StrictMacro {
         `!mpbelow ${mpCost($skill`Stuffed Mortar Shell`)}`,
         Macro.trySkill($skill`Stuffed Mortar Shell`)
       );
-    return (useCinch ? macroHead.trySkill($skill`Cincho: Confetti Extravaganza`) : macroHead)
-      .while_(
-        `!mpbelow ${mpCost($skill`Saucegeyser`)} && hasskill ${toInt($skill`Saucegeyser`)}`,
-        Macro.skill($skill`Saucegeyser`)
-      )
-      .while_(
-        `!mpbelow ${mpCost($skill`Saucestorm`)} && hasskill ${toInt($skill`Saucestorm`)}`,
-        Macro.skill($skill`Saucestorm`)
-      )
-      .attack()
-      .repeat();
+    if (myPrimestat() === $stat`Mysticality`) {
```

Comment:
We defined `mainStat` above, so there's no reason to call `myPrimestat()` here. Alternatively, we should just use `myPrimestat()` wherever we need it and not need to define and export `mainStat` above